prowler/lib/outputs/finding.py

```diff
@@ -111,6 +111,18 @@ def generate_output(
             finding_output (Finding): the finding output object
 
         """
+        if not check_output.resource_id:
```

Comment:
Why not to always at the end of this function when the output is unified?

Reply:
Done, good idea!